package.json

```diff
@@ -1,6 +1,9 @@
 {
   "name": "@solid/oidc-op",
-  "version": "0.5.0",
+  "version": "0.5.0",,
```

Comment:
```suggestion
  "version": "0.5.0",
```

Reply:
Hehe, yeah, was just fixing that, and then it turned out my Emacs no longer worked, cause I made the mistake of upgrading to Mac OS Catalina without looking into it! :) So I had to re-set that up.